content/ja/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,222 @@
+バージョン1.4以降、Nodeコントローラは、Podの退役について決定する際に、クラスタ内のすべてのNodeの状態を調べます。
+
+ほとんどの場合、排除の速度は1秒あたり`--node-eviction-rate`に設定された数値（デフォルトは秒間0.1）です。つまり、10秒間に1つ以上のPodをNodeから追い出すことはありません。
+
+特定のアベイラビリティーゾーン内のNodeのステータスが異常になると、Node排除の挙動が変わります。Nodeコントローラは、ゾーン内のNodeの何%が異常（NodeReady条件がConditionUnknownまたはConditionFalseである）であるかを同時に確認します。
+異常なNodeの割合が少なくとも `--healthy-zone-threshold`に設定した値を下回る場合（デフォルトは0.55）であれば、退役率は低下します。クラスタが小さい場合（すなわち、 `--large-cluster-size-threshold`の設定値よりもNode数が少ない場合。デフォルトは50）、退役は停止し、そうでない場合、退役率は秒間で`--secondary-node-eviction-rate`の設定値（デフォルトは0.01）に減少します。
+これらのポリシーがアベイラビリティーゾーンごとに実装されているのは、1つのアベイラビリティーゾーンがマスターから分割される一方、他のアベイラビリティーゾーンは接続されたままになる可能性があるためです。
+クラスターが複数のクラウドプロバイダーのアベイラビリティーゾーンにまたがっていない場合、アベイラビリティーゾーンは1つだけです（クラスター全体）。
+
+Nodeを複数のアベイラビリティゾーンに分散させる主な理由は、1つのゾーン全体が停止したときにワークロードを正常なゾーンに移動できることです。
+したがって、ゾーン内のすべてのNodeが異常である場合、Nodeコントローラは通常のレート `--node-eviction-rate`で退役します。
+コーナーケースは、すべてのゾーンが完全に不健康である（すなわち、クラスタ内に健全なNodeがない）場合です。
+このような場合、Nodeコントローラはマスター接続に問題があると見なし、接続が回復するまですべての退役を停止します。
+
+Kubernetes 1.6以降では、Nodeコントローラーは、Podがtaintを許容しない場合、 `NoExecute`のtaintを持つNode上で実行されているPodを排除する責務もあります。
+さらに、デフォルトで無効になっているアルファ機能として、NodeコントローラーはNodeに到達できない、または準備ができていないなどのNodeの問題に対応するtaintを追加する責務があります。
+`NoExecute`のtaint及び上述のアルファ機能に関する詳細は、[こちらのドキュメント](/docs/concepts/configuration/taint-and-toleration/)をご覧ください。
+
+バージョン1.8以降、Nodeコントローラに対してNodeの状態を表すtaintを作成する責務を持たせることができます。これはバージョン1.8のアルファ機能です。
+
+### Nodeの自己登録
+
+kubeletのフラグ `--register-node`がtrue（デフォルト）のとき、kubeletは自分自身をAPIサーバーに登録しようとします。これはほとんどのディストリビューションで使用されている推奨パターンです。
+
+自己登録については、kubeletは以下のオプションを伴って起動されます:
+
+  - `--kubeconfig` - 自分自身をAPIサーバーに対して認証するための資格情報へのパス
+  - `--cloud-provider` - 自身に関するメタデータを読むためにクラウドプロバイダーと会話する方法
+  - `--register-node` - 自身をAPIサーバーに自動的に登録
+  - `--register-with-taints` - 与えられたtaintのリストでNodeを登録します (カンマ区切りの `<key>=<value>:<effect>`). `register-node`がfalseの場合、このオプションは機能しません
+  - `--node-ip` - NodeのIPアドレス
+  - `--node-labels` - Nodeをクラスターに登録するときに追加するラベル（1.13以降の[NodeRestriction許可プラグイン](/docs/reference/access-authn-authz/admission-controllers/#noderestriction)によって適用されるラベルの制限を参照）
+  - `--node-status-update-frequency` - kubeletがNodeのステータスをマスターにPOSTする頻度の指定
+
+[Node認証モード](/docs/reference/access-authn-authz/node/)および[NodeRestriction許可プラグイン]/docs/reference/access-authn-authz/admission-controllers/#noderestriction)が有効になっている場合、kubeletは自分自身のNodeリソースを作成/変更することのみ許可されています。
+
+#### 手動によるNode管理 {#manual-node-administration}
+
+クラスター管理者はNodeオブジェクトを作成および変更できます。
+
+管理者が手動でNodeオブジェクトを作成したい場合は、kubeletフラグ `--register-node = false`を設定してください。
+
+管理者は`--register-node`の設定に関係なくNodeリソースを変更することができます。
+変更には、Nodeにラベルを設定し、それをunschedulableとしてマークすることが含まれます。
+
+Node上のラベルは、スケジューリングを制御するためにPod上のNodeセレクタと組み合わせて使用できます。
+例えば、PodをNodeのサブセットでのみ実行する資格があるように制限します。
+
+Nodeをunschedulableとしてマークすると、新しいPodがそのNodeにスケジュールされるのを防ぎますが、Node上の既存のPodには影響しません。
+これは、Nodeの再起動などの前の準備ステップとして役立ちます。たとえば、Nodeにスケジュール不可能のマークを付けるには、次のコマンドを実行します:
+
+```shell
+kubectl cordon $NODENAME
```

Comment:
If `<insert-node-name-here>` was localized above, that'd be appropriate here too I think.